### PR TITLE
merging feature/signup page to develop

### PR DESCRIPTION
**Pull request by Andela-thomas**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ var gulp = require('gulp'),
   imagemin = require('gulp-imagemin'),
   nodemon = require('gulp-nodemon'),
   karma = require('gulp-karma'),
+  protractor = require('protractor'),
```
Comment (andela-gukpere) [inline gulpfile.js]: @Andela-thomas How tho?

Comment (Andela-thomas) [inline gulpfile.js]: i am trying to figure that out,

**Comment by andela-gukpere** [comment]
Before committing the changes, did you test the test on your local machine?
I doubt, but I hope you did.

**Comment by Andela-thomas** [comment]
i have it work perfectly

**Comment by andela-gukpere** [comment]
Are you sure?
screen shot 2015-10-08 at 3 30 13 pm

**Comment by andela-gukpere** [comment]
@Andela-thomas @andela-nmartin 

The _protractor config_ references a selenium drive **2.44.0**, and we install version **2.47.1** in our _package.json_

**Solution**

Have package.json install a fixed version, which matches the `.jar` file referenced in the protractor config.
See the screenshots below for reference.

####package.json
screen shot 2015-10-08 at 3 33 07 pm

####protractor.config.js
screen shot 2015-10-08 at 3 32 48 pm

**Comment by Andela-thomas** [comment]
Hello, Godson

OK,  i will fix it in tomorrow morning thanks for tips.

with regards

Nyambati Thomas | Software Developer

(K) +254 711150345  | http://andela.com

Join Andela. Discover Andela: TechCrunch, USNews, & Wired

**Pull request closed without merging**